### PR TITLE
Adding CANCEL button in the footer of Modals

### PR DESCRIPTION
**Pull request by mohammed2raja**
Adding a CANCEL button in the footer of Modals

### CODE REVIEW

File: views/dialog.hbs [hunk 1]
```diff
@@ -10,12 +10,12 @@
     {{/if}}
     <div class="modal-body"><p>{{text}}</p></div>
     <div class="modal-footer">
+      <button type="button" class="btn btn-link" data-dismiss="modal">{{cancelBtnLabel}}</button>
```
Comment (bzalasky) [inline views/dialog.hbs]: Getting closer, but why isn't the cancel button included in the array of buttons rendered just below this?

File: views/progress-dialog.hbs [hunk 2]
```diff
@@ -26,13 +26,13 @@
 
 {{#*inline "footer"}}
   <div class="modal-footer">
+    <button type="button" class="btn btn-link" data-dismiss="modal">{{cancelBtnLabel}}</button>
```
Comment (bzalasky) [inline views/progress-dialog.hbs]: Ditto.

File: views/dialog.hbs (hunk 1 above)
Comment (mohammed2raja) [reply views/dialog.hbs]: https://github.com/lookout/gringotts/blob/master/views/dialog-view.coffee#L9
This "Button" array gets override when we send a "button" object while extending this class.
I tried it but it does not work.

Comment (eugenet8k) [reply views/dialog.hbs]: Yeah all the buttons In this dialog should be rendered based on array in the class instance.

File: views/progress-dialog.hbs (hunk 2 above)
Comment (eugenet8k) [reply views/progress-dialog.hbs]: Cancel is no better than other buttons. It shouldn’t have any special treatment like a label prop in template data.

**Pull request closed without merging**